specifyweb/frontend/js_src/lib/components/WorkBench/WbSpreadsheet.tsx

```diff
@@ -129,14 +133,14 @@ function WbSpreadsheetComponent({
               } as const)
             : ({
                 row_above: {
-                  disabled: () => isReadOnly,
+                  disabled: () => isReadOnly || dataset.isupdate,
                 },
                 row_below: {
-                  disabled: () => isReadOnly,
+                  disabled: () => isReadOnly || dataset.isupdate,
                 },
                 remove_row: {
                   disabled: () => {
-                    if (isReadOnly) return true;
+                    if (isReadOnly || dataset.isupdate) return true;
```

Comment:
Changes look good!

For this one, are you sure removing also needs to be disabled? It would be beneficial to allow removing rows, because it is possible that those rows might have some errors (validation ones) but you still want the other rows to be updated. In the current case, if we do not allow removal, then the user would have to make the batch-edit query again and reapply changes. 
+ we check for versions and do not allow updates if the version differs. So, it is definitely possible that the user simply wants to remove the row with the changed version.